roles/zuul/defaults/main.yml

```diff
@@ -12,6 +12,7 @@ zuul_logserver_fqdn: "logs.example.org"
 zuul_webserver_fqdn: "zuul.example.org"
 zuul_webserver_admin: "admin@example.org"
 zuul_zookeeper_fqdn: "zookeeper01.example.org"
+zuul_use_acme: false
```

Comment:
This var should be mentioned in the README.rst

Ideally an example how to set this up in doc/source/zuul would also be nice, but that can be done as follow-up.